R/predict.ergm.R

```diff
@@ -0,0 +1,75 @@
+#' ERGM-based conditional tie probabilities
+#'
+#' This will calculate **conditional** tie probabilities for all dyads in `net`
+#' based on the ERGM model `fit`.
+#'
+#' @param fit fitted ERGM model object
+#' @param net network object
+#'
+#' @return 
+#' N x N matrix of conditional tie probabilities, where N is the number of nodes in `net`.
+#' 
+#' @method predict ergm
+#'
+#' @export
+
+predict.ergm <- function(fit, net) {
```

Comment:
The first argument should probably be `object` for consistency with `stats::predict()` and have a `...` argument.

Reply:
Yeah, I had it as `cond_probs` and renamed it to `predict` just before pushing.